redis.conf

```diff
@@ -0,0 +1,8 @@
+port              6379
```

Comment:
Can we add a comment in this file? If yes, please add a comment asking developers to keep these values in sync with the REDIS_PORT constant.